src/create-tree.ts

```diff
@@ -20,6 +21,8 @@ export async function createTree(
   for (const path of Object.keys(changes.files)) {
     const value = changes.files[path];
 
+    // TODO use Symbol here which will result in a breaking change
+    // if (value === deleteFile) {
```

Comment:
or we keep it as is at the drawback of having inconsistency between return values from an update function vs this code snipped

```
files: {
  "file/to/delete.txt": null
}

```

Reply:
I'd rather go with a breaking change

Reply:
Done. Documented in the readme and commit message